krabs/krabs/trace.hpp

```diff
@@ -245,6 +273,22 @@ namespace krabs {
         manager.stop();
     }
 
+    template <typename T>
+    EVENT_TRACE_LOGFILE trace<T>::open()
+    {
+        eventsHandled_ = 0;
```

Comment:
Is this in a scenario where you're doing a re-open? I'm unclear why only this variable needs to be reset?

Reply:
I tried to provide functional equivalence to what start() does today. In essence, this PR provides open(); process(); as an alternative to start(). Since start() clears the eventsHandled_ counter, I did the same in open().